web_src/css/base.css

```diff
@@ -989,14 +989,6 @@ table th[data-sortt-desc] .svg {
   box-shadow: 0 0 0 1px var(--color-secondary) inset;
 }
 
-.emoji {
-  font-size: 1.25em;
-  line-height: var(--line-height-default);
-  font-style: normal !important;
-  font-weight: var(--font-weight-normal) !important;
-  vertical-align: -0.075em;
-}
-
 .emoji img {
```

Comment:
This is targeting fallback images, should probably remove these fallback images in 1.24 or later, we can require browsers to have emoji support at this point in time.

Reply:
I guess no, see the newly added comment

Reply:
Ah you are right, we don't support fallbacks, and this is only for the custom image emojis.